app/models/job_invocation.rb

```diff
@@ -15,7 +15,8 @@ class JobInvocation < ActiveRecord::Base
   validates :job_name, :presence => true
   validates_associated :targeting, :template_invocations
 
-  scoped_search :on => :job_name
+  scoped_search :on => :job_name, :complete_value => true
+  scoped_search :on => :description, :complete_value => true
```

Comment:
I'm not sure whether allowing autocomplete on description is a good idea since it does not escape quotes and searching fails because of it. See [here](https://imgur.com/a/fOwjW).